### PR TITLE
Add documentation on autoreloading of file.properties

### PR DESCRIPTION
**Pull request by sdkottegoda**
Documentation for https://github.com/wso2/micro-integrator/issues/2332

### CODE REVIEW

File: en/docs/reference/mediators/property-reference/accessing-properties-with-xpath.md [hunk 1]
```diff
@@ -200,7 +200,9 @@ Syntax:
 
 #### file scope
 
-You can retrieve properties defined in the `file.properties` configuration file using the following syntax.
+You can retrieve properties defined in the `file.properties` configuration file using the following syntax. Properties 
+in the file are reloaded periodically according to the time interval defined by the `file.properties.sync.interval` 
+System property in Seconds.
```
Comment (Mariangela) [inline en/docs/reference/mediators/property-reference/accessing-properties-with-xpath.md:205]:
```suggestion
system property in seconds.
```

**Review by Mariangela — APPROVED** [review]

**Comment by Mariangela** [comment]
@sdkottegoda, please make sure to sign the [Contributor License Agreement (CLA)](https://cla-assistant.io/wso2/docs-apim?pullRequest=4252)

**Comment by Mariangela** [comment]
@sdkottegoda , pls send this fix to the 4.0.0 branch as well

**Comment by sdkottegoda** [comment]
> @sdkottegoda, please make sure to sign the [Contributor License Agreement (CLA)](https://cla-assistant.io/wso2/docs-apim?pullRequest=4252)

I have signed, but it shows me as not signed

**Comment by sdkottegoda** [comment]
> @sdkottegoda , pls send this fix to the 4.0.0 branch as well

This is related to the public fix which was shipped as a wum update for MI-1.2.0. And the fix is not ported to 4.0.0 yet, hence not applicable.

**Pull request merged**